### DISCUSSION TITLE
Banding in Test Image -- Normal?

### DISCUSSION THREAD

**Discussion by ldb20151** [aaronwmorris/indi-allsky#1729] [Q&A] [answered]
I'm waiting for various seals to arrive before I can take my newly assembled (almost assembled) camera outside for true first light.  In the mean time, I've been exploring the configuration on the workbench.  I've let it run several days with room lighting on, so its very well lit, with camera pointing at a clock as the subject.  The first thing I noticed is some pretty serious 'banding'.  I can't seem to affect this with any of the obvious settings.  The picture here was taken with the camera in default config settings.  Though present most of the time, the  banding isn't always seen.  The only variable I can think of that might affect it would be the level of room lighting -- slightly dimmer in pictures without banding and perhaps a slight variation in the camera position.  Settings were not changed.  

I read discussions of a similar effect caused by bright lights like sun/moon/street lights in the field of view, but all of the examples I've seen were circular bands vs my linear bands.  Is this normal?  
![image](https://github.com/user-attachments/assets/add02146-5067-43af-ae71-f99431976d63)

**Comment by aaronwmorris** [comment, marked as answer]
Those look like florescent lights.  This banding is probably due to the inherent high frequency flicker of the ballast.  Most cameras operate with a rolling shutter, so you are seeing the flicker via the rolling shutter.  Once you get the camera outside, or under a different light, it should go away.

**Reply by ldb20151** [reply]
The lights are actually all LED, but I agree the effect is very similar to what would be expected from fluorescent tubes or cfls.  Maybe the LEDs are still oscillating at the line frequency?  Anyway, I confirmed the issue goes away when the lights are off (indirect ambient daylight only).  System must be OK.  Thx.
